components/LayoutWithStatusBanner.tsx

```diff
@@ -0,0 +1,62 @@
+"use client";
+
+import { usePathname } from "next/navigation";
+import "@/app/globals.css";
+import StatusBanner from "@/components/StatusBanner";
+import { Button } from "@/components/ui/button";
+
+interface LayoutWithStatusBannerProps {
+  children: React.ReactNode;
+  issues: any[];
+}
+
+export default function LayoutWithStatusBanner({
+  children,
+  issues,
+}: LayoutWithStatusBannerProps) {
+  const pathname = usePathname();
+  const showStatusBanner = pathname === "/";
+  const repoNames = issues.map((repo) => repo.name).join(", ");
+
+  return (
+    <div>
+      {showStatusBanner && (
```

Comment:
So when there isn't a status banner we have an empty div?

Reply:
changed so when there are no open issues, the status banner becomes green